src/pykeen/datasets/wk3l.py

```diff
@@ -0,0 +1,125 @@
+# -*- coding: utf-8 -*-
+
+"""The Wk3l-15k dataset family.
+
+Get a summary with ``python -m pykeen.datasets.wk3l``
+"""
+import logging
+import pathlib
+import zipfile
+from typing import Optional, Tuple, cast
+
+import click
+import pandas
+from more_click import verbose_option
+from pystow.utils import download_from_google
+
+from pykeen.datasets import LazyDataset
+from pykeen.triples import TriplesFactory
+from pykeen.typing import TorchRandomHint
+
+__all__ = [
+    "WK3l15k",
+]
+
+logger = logging.getLogger(__name__)
+
+GOOGLE_DRIVE_ID = "1AsPPU4ka1Rc9u-XYMGWtvV65hF3egi0z"
+GRAPH_PAIRS = ("en_fr", "en_de")
+SIDES = tuple(sum((pair.split("_") for pair in GRAPH_PAIRS), start=[]))
+
+
+class WK3l15k(LazyDataset):
+    """The WK3l-15k dataset family.
+
+    The datasets were first described in https://www.ijcai.org/Proceedings/2017/0209.pdf.
+    """
+
+    ratios = (0.8, 0.1, 0.1)
+
+    def __init__(
+        self,
+        graph_pair: str = "en_de",
+        side: str = "en",
+        cache_root: Optional[str] = None,
+        eager: bool = False,
+        create_inverse_triples: bool = False,
+        random_state: TorchRandomHint = None,
+        force: bool = False,
+    ):
+        if graph_pair not in GRAPH_PAIRS:
+            raise ValueError(f"Invalid graph pair: Allowed are: {GRAPH_PAIRS}")
+        available_sides = graph_pair.split("_")
+        if side not in available_sides:
+            raise ValueError(f"side must be one of {available_sides}")
+        suffix = 5 if graph_pair == "en_fr" else 6
+        file_name = f"P_{side}_v{suffix}.csv"
+        self._relative_path = pathlib.PurePosixPath("data", "WK3l-15k", graph_pair, file_name)
+        self.cache_root = self._help_cache(cache_root)
+
+        self.random_state = random_state
+        self.drive_id = GOOGLE_DRIVE_ID
+        self.name = "wk3l15k.zip"
```

Comment:
@cthoyt the same zip file contains two more dataset families: WK3l-120k (a larger version of this dataset), and CN3l, a similar dataset extracted from conceptnet.

Reply:
cool, so which one you use can be configured?